modules/instance/versions.tf

```diff
@@ -1,27 +1,9 @@
-#####################################################
-# Instance Module
-# Copyright 2020 IBM
-#####################################################
-
-/***************************************************
-NOTE: To source a particular version of IBM terraform provider, configure the parameter `version` as follows
 terraform {
   required_version = ">=0.13"
```

Comment:
suggest to update this to `>= 1.3.0` to be consistent with other modules

Reply:
updated